baselines/Google Classroom Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -175,9 +175,43 @@ Only teachers SHALL be allowed to unenroll students from classes.
 ### Implementation
 To configure the settings for Student Unenrollment:
 
-#### GWS.CLASSROOM.4.1v0.2 Instructions
+#### GWS.CLASSROOM.5.1v0.2 Instructions
 1.  Sign in to the [Google Admin Console](https://admin.google.com).
 2.  Select **Apps** -\> **Additional Google Service** -\> **Classroom**.
 3.  Select **Student unenrollment**.
 4.  Select **Teachers Only**.
 5.  Select **Save**.
+
+## 5. Class Creation
+
+This section covers who has the ability to create classes.
+
+### Policy
+
+#### GWS.CLASSROOM.5.1v0.2
+Who can create classes SHALL be set to Verified teachers only.
+
+- _Rationale:_ Allowing pending teachers to create classes allows for potential unauthorized data creation which creates a security risk. By allowing only verified teachers to create classes mitigates the risk.
```

Comment:
I agree that there's a risk here, but I don't think "unauthorized data creation" exactly covers it. If I impersonate a teacher and create a class and some users join it, can I, as a pending teacher, see the PII of those students? It looks like I'd be able to see those students' [names, photos, and email addresses](https://support.google.com/edu/classroom/answer/6386395?hl=en#:~:text=Who%20can%20see%20what%3F&text=Teachers%20in%20your%20classes%20can,view%20your%20name%20and%20photo.). If nothing else, me creating a class as an unverified teacher creates an unfounded trust relationship between me and any students that join my class that I might be able to exploit, e.g., for phishing. These are what I see as the true risks here.

```suggestion
- _Rationale:_ Allowing pending teachers to create classes potentially allows students to impersonate teachers and exploit the trusted relationship between teacher and student, e.g., to phish sensitive information from the students. Restricting class creation to verified teachers reduces this risk.
```